modules/ROOT/pages/installation/server.adoc

```diff
@@ -312,8 +327,7 @@ If the NOM Server is required to support self-registered agents, then additional
 | `grpc.server.security.clientAuth`
 | `GRPC_SERVER_SECURITY_CLIENT_AUTH`
 | Option to indicate whether to verify client certificates. 
-Required for agent self-registration. 
-Possible values: `OPTIONAL` to simultaneously allow manual registration or `REQUIRE` to only allow mutually authenticated agents.
```

Comment:
Yes, with security fixes unauthenticated flows were removed and it's no longer possible to connect with an agent without auth for approval. So mTLS is default and `GRPC_SERVER_SECURITY_CLIENT_AUTH` should always be `OPTIONAL` to allow other clients to connect.

Reply:
Is there then a difference between `OPTIONAL` and not set? 

If not, we should get rid of it as a configuration value and fix it in `application.properties` to be `OPTIONAL` instead. 

As a user, I am confused by these docs, where something is optional, has an example value `OPTIONAL` and this is its only value it can have.

![image](https://github.com/neo4j/docs-ops-manager/assets/2542670/612561fc-9c2d-4275-b9f1-73df06589702)

Reply:
I guess it no longer makes any sense as a configuration value then, will remove